transports/janus_rabbitmq.c

```diff
@@ -945,8 +950,14 @@ void *janus_rmq_in_thread(void *data) {
 				JANUS_LOG(LOG_VERB, "Trying to reconnect with RabbitMQ Server\n");
 				int result = janus_rabbitmq_connect();
 				if(result < 0) {
-					JANUS_LOG(LOG_ERR, "Failed to reconnect to RabbitMQ Server. Retrying in 5s...\n");
-					g_usleep(5000000);
+					JANUS_LOG(LOG_WARN, "Failed to reconnect to RabbitMQ Server. Retrying in %fs...\n", (gfloat)rmq_reconnect_backoff/1000000);
+					g_usleep(rmq_reconnect_backoff);
+					rmq_reconnect_backoff *= rmq_reconnect_backoff_multiplier;
+					if (rmq_reconnect_backoff >= rmq_reconnect_backoff_max) {
+						rmq_reconnect_backoff = rmq_reconnect_backoff_max;
+					}
```

Comment:
Code style: no space between `if` and bracket. Besides, I think you can omit the curly brackets, as the action is simple.